### ISSUE TITLE
How can I tell if the extension is working?

### ISSUE THREAD

**Issue by ibayer** [raulpardo/thunderbird-langtool-mailextension#1]
Hi,

thanks for making this extension available. I find the idea of having my local langtool server accessible in Thunderbird very attractive.

After installing the extension if found the following addition in my Thunderbird. 

![image](https://user-images.githubusercontent.com/1497140/214154206-9d0ba145-2923-42b6-bad5-e697847a8456.png)

Seems like it doesn't do much with my test e-mail. I have used the remote langtool server for testing.

Can you share some screenshot showing the extension in action? It would be great if the extension could directly integrate into the text, like the regular spellchecker does, but I'm not sure if this is even possible with an extension.

I'm currently using a vs code extension https://marketplace.visualstudio.com/items?itemName=valentjn.vscode-ltex which does provide this spellchecker like integration.

**Comment by raulpardo** [comment]
Hi @ibayer!

Sorry for the late reply. I am happy that you tried the extension!

My first guess is that you are editing an HTML email. Is this the case? If so, the problem is that the extension does not support HTML emails yet. You can try to edit a plain text email and check again whether the extension works. If not, please let me know.

Here is a screenshot on how it should look like:
![image](https://user-images.githubusercontent.com/8382285/219880680-c599ee32-6b11-408b-be24-72a30fc8338f.png)





I should also point out that LanguageTool has now an official Thunderbird add-on (https://addons.thunderbird.net/en-US/thunderbird/addon/grammar-and-spell-checker/). Their interface is way better than the one in this add-on. However, I still find a plus on using this add-on as opposed to the official one. The LanguageTool add-on requires permission for 1) Access your data for all websites, 2) Read and modify browser settings and 3) Access browser tabs. This add-on only requires permission for 1) Read and modify your email messages as you compose and send them (you can check this in the permissions tab when you install the add-on). I presume that the nice interface in the LanguageTool add-on requires those permissions for implementing the interface.

**Comment by fedeinthemix** [comment]
I've tried this as well and also get no output even for text emails
![image](https://user-images.githubusercontent.com/7670450/225691669-e0fe0910-31b7-4ab0-86b7-703ccdfade2c.png)
I'm using Thunderbird 102.8.0

**Comment by raulpardo** [comment]
Hmm this is strange. What server did you specify? Are you running your own Langtool server?

**Comment by fedeinthemix** [comment]
I run a local Langtool server, but I also tried with the public one.

**Comment by raulpardo** [comment]
I see, from the screenshot it looks like you are getting an empty reply for the server. Can you test that you get a valid answer from the server by pasting this URL (http://localhost:8081/v2/check?language=en-US&text=my+text) in a web browser, and verifying that you get JSON file? (this URL assumes that you are running a local instance at http://localhost:8081/)

**Comment by fedeinthemix** [comment]
He're's part of the output
![image](https://github.com/user-attachments/assets/fd2e34c9-aaf1-445b-ac83-8cff4187d401)

**Comment by raulpardo** [comment]
This looks good. The server is working properly. In the screenshot here https://github.com/raulpardo/thunderbird-langtool-mailextension/issues/1#issuecomment-1472334556, did you have a some text in the email composing window?

**Comment by fedeinthemix** [comment]
Yes, the server works fine. I use it with the official add-on without problems.

I've checked this add-on by typing some text with errors and waited a bit.

**Comment by raulpardo** [comment]
Hmm, then I am not sure what the problem is. Did you set in the correct URL in the preferences of the extension?
